src/nvim/tui/tui.c

```diff
@@ -1009,17 +1012,51 @@ static void tui_mouse_on(UI *ui)
 {
   TUIData *data = ui->data;
   if (!data->mouse_enabled) {
+#ifdef WIN32
+    const char *term = os_getenv("TERM");
+    bool toggle_uv_vterm_state =
```

Comment:
what's a more meaningful name for this? maybe `no_vti` ?
```suggestion
    bool no_vti =
```